### ISSUE TITLE
Pricing page polish

### ISSUE THREAD

**Issue by agjohnson** [readthedocs/site-community#167]
A few points that I accumulated for a second pass eventually:

- [ ] Hide feature list table on mobile, it looks awful stacked
- [ ] Set a min-height on the community/commercial plans tabs so that page content doesn't jump when switching tabs
- [ ] Linking to either community/commercial tab should also focus the tab. Currently, it just switches the tab and doesn't scroll up

**Comment by humitos** [comment]
I'm not sure if this effect is on purpose or not, so, I'm commenting here.

When I click on the "Pricing" link, it goes to the pricing page and _automatically scrolls down to the pricing table_. Which I find a little "aggressive" (not sure if that's the correct word) --but I _personally_ don't like the page moving it automatically by itself.

![Peek 2022-11-16 10-14](https://user-images.githubusercontent.com/244656/202139174-8257773b-be96-4312-ab8b-17bcf0ad89f1.gif)

**Comment by agjohnson** [comment]
Ah good catch, this is a regression from earlier on in my work. It's not intended, the JS that controls this probably should track user actions more aggressively. I'll treat this issue as a bug and get a fix out

**Comment by agjohnson** [comment]
Yeah, this would be possible, it waffled on putting this in the first iteration. It would be a good addition, but might take some custom code. I'll note this above though :+1:
